_install-and-configure/plugins.md

```diff
@@ -205,6 +205,20 @@ Continue with installation? [y/N]y
 Restart your OpenSearch node after installing a plugin.
 {: .note}
 
+### Installing multiple plugins
+
+Multiple plugins can be installed in one invocation
+
+#### Usage:
```

Comment:
```suggestion
#### Usage
```

Reply:
When I looked at the other docs, they were putting `:` on the back.
Is it okay to remove this?

Reply:
https://github.com/opensearch-project/documentation-website/blob/9d3a738b218250ea912a401f7cdf0a6c8189729b/_install-and-configure/plugins.md?plain=1#L35

like this